### PR TITLE
WINC-797: Add subscription annotation to CSV

### PR DESCRIPTION
**Pull request by alinaryan**
This PR adds a subscription annotation to our CSV to prepare for the new
gating validation being added by CVP. We will be unable to release without
it starting in April.

### CODE REVIEW

**Comment by mansikulkarni96** [comment]
LGTM

**Comment by saifshaikh48** [comment]
/hold

Please make the change in the base CSV and run `make build` for the bundle changes

File: bundle/manifests/windows-machine-config-operator_v1_serviceaccount.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
```
Comment (mansikulkarni96) [inline bundle/manifests/windows-machine-config-operator_v1_serviceaccount.yaml:1]: we cannot add service account file in bundle manifest, it causes issues with duplicate SA bundle validation in upgrade case.

File: config/manifests/bases/windows-machine-config-operator.clusterserviceversion.yaml [hunk 2]
```diff
@@ -11,6 +11,7 @@ metadata:
     olm.skipRange: '>=4.0.0 <5.0.0'
```
Comment (jrvaldes) [inline config/manifests/bases/windows-machine-config-operator.clusterserviceversion.yaml:11]: @alinaryan , consider squashing the commits, no need for two separated commits.

Comment (alinaryan) [reply config/manifests/bases/windows-machine-config-operator.clusterserviceversion.yaml:11]: squashed

**Comment by selansen** [comment]
do we need to backport to community release?

**Comment by alinaryan** [comment]
>do we need to backport to community release?

No, this is not needed for community.

**Pull request merged**